cnd/src/db/models.rs

```diff
@@ -0,0 +1,39 @@
+use crate::db::schema::swaps;
+use diesel::Queryable;
+
+#[derive(Queryable, Debug)]
+pub struct Swap {
+    pub swap_id: String,
+    pub alpha_ledger: String,
+    pub beta_ledger: String,
+    pub alpha_asset: String,
+    pub beta_asset: String,
+    pub role: String,
+    pub counterparty: i32, // FIXME: this should be u64 but cannot work out how to deserialize it?
```

Comment:
need to implement a manual deserialize trait, not sure you can do that with `Queryable`.